syncthing/manifest.json

```diff
@@ -21,8 +21,8 @@
   "author": {
     "support_email": "Alexander@Bushnev.pro",
     "homepage": "https://github.com/DataDog/integrations-extras",
-    "sales_email": "Alexander@Bushnev.pro",
-    "name": "unknown"
+    "sales_email": "Alexander@Bushnev.ru",
```

Comment:
Is this change intentional?

Reply:
Looks like a merge conflict when i merged in from master this afternoon. I'll update so it matches what exists in the repo currently.